package.json

```diff
@@ -59,20 +59,21 @@
     "check-updates": "ncu -u && ncu -u --cwd examples/js"
   },
   "dependencies": {
-    "@feathersjs/authentication": "^4.5.15",
-    "@feathersjs/authentication-client": "^4.5.15",
-    "@feathersjs/authentication-local": "^4.5.15",
-    "@feathersjs/errors": "^4.5.15",
+    "@feathersjs/authentication": "^5.0.0-pre.19",
```

Comment:
Please update all dependencies to pre.29 or even pre.30

Reply:
Updated to pre.31